packages/react-composites/tests/browser/call/hermetic/ParticipantPane.test.ts

```diff
@@ -7,16 +7,15 @@ import {
   dataUiId,
   hidePiPiP,
   isTestProfileDesktop,
-  isTestProfileStableFlavor,
   pageClick,
   stableScreenshot,
   waitForSelector
 } from '../../common/utils';
 import { buildUrlWithMockAdapter, defaultMockCallAdapterState, defaultMockRemoteParticipant, test } from './fixture';
 
 test.describe('Participant pane tests', async () => {
+  /* @conditional-compile-remove(one-to-n-calling) */
```

Comment:
People Pane should actually be conditionally compiled for both PSTN and one-to-n-calling.

Reply:
I will go back and add the missing conditional compilation for PeoplePane related code. But in the meantime please also add PSTN here :)

Reply:
https://github.com/Azure/communication-ui-library/pull/2493 for tracking